extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -100,40 +100,48 @@ const CreateActionContent = () => {
         <Instruction>
           <InstructionTitle>{formatMessage('Step 2')}</InstructionTitle>
           <InstructionDetails>
-            <Text>
-              {formatMessage(
-                'Select tenant and subscription, enter resource group name and resource name, and select region.'
-              )}
-            </Text>
+            <Text>{formatMessage('Select Azure subscription and resource group name')}</Text>
           </InstructionDetails>
         </Instruction>
         <Instruction>
           <InstructionTitle>{formatMessage('Step 3')}</InstructionTitle>
           <InstructionDetails>
-            <Text>
-              {formatMessage(
-                'Review and create new resources. Once provisioned these resources will be available in your Azure portal.'
-              )}
-            </Text>
+            <Text>{formatMessage('Select and name your new resources')}</Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 4')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>{formatMessage('Review and confirm resources to be created')}</Text>
           </InstructionDetails>
         </Instruction>
       </Details>
+      <Summary>
+        <Text>{formatMessage('Once provisioned, your new resources will be available in the Azure portal. ')}</Text>
```

Comment:
We shouldn't add spaces to the end of `formatMessage` to add space between links - it could easily get lost in translation. We should either add a space with `{' '}` or use `formatMessage.rich` take a look at https://github.com/microsoft/BotFramework-Composer/blob/b45b4c8a06d513285c289efcd70ddda1ee30e009/Composer/packages/client/src/pages/botProject/AllowedCallers.tsx#L149

Reply:
I would add &nbsp; between </Text> and <LearnMoreLink>.

Reply:
What's the benefit of `&nbsp;` over `{' '}`? Since the link is not nested in the text I think we should use either `&nbsp;` or `{' '}` to add the space and not `formatMessage.rich`

Reply:
`&nbsp;` is standard HTML, the `{' '}` requires running code and context switching from HTML to JS.  It doesn't make much difference.